pkg/msgconv/wa-misc.go

```diff
@@ -244,3 +244,24 @@ func (mc *MessageConverter) convertKeepInChatMessage(ctx context.Context, msg *w
 		},
 	}, nil
 }
+
+func (mc *MessageConverter) convertRichResponseMessage(ctx context.Context, msg *waE2E.AIRichResponseMessage) (*bridgev2.ConvertedMessagePart, *waE2E.ContextInfo) {
+	var body strings.Builder
+
+	for i, submsg := range msg.GetSubmessages() {
+		if submsg.GetMessageType() == waE2E.AIRichResponseMessage_AI_RICH_RESPONSE_TEXT {
```

Comment:
todo: will support more types here

Reply:
a `switch` will be more appropriate when adding more types